Makefile

```diff
@@ -41,22 +40,22 @@ licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
 	golicense hack/license-config.hcl karpenter
 
 apply: ## Deploy the controller into your ~/.kube/config cluster
-	$(WITH_GOFLAGS) ko apply -B -f config
+	helm template karpenter ./config/ | $(WITH_GOFLAGS) ko apply -B -f -
 
 delete: ## Delete the controller from your ~/.kube/config cluster
-	ko delete -f config
+	helm template karpenter ./config/ |  ko delete -f -
 
 codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 	./hack/codegen.sh
 
 publish: ## Generate release manifests and publish a versioned container image.
-	@aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin $(RELEASE_REPO)
-	$(WITH_RELEASE_REPO) $(WITH_GOFLAGS) ko resolve -B -t $(RELEASE_VERSION) --platform all -f config > $(RELEASE_MANIFEST)
-
-helm: ## Generate Helm Chart
-	cp $(RELEASE_MANIFEST) charts/karpenter/templates
+	@aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $(RELEASE_REPO)
+	@cp config/templates/* charts/karpenter/templates
```

Comment:
I'm a little confused here. Why does anything need to be copied? I thought the point was that all helm/config are now in the same location.

Reply:
Sorry it was a misunderstanding, I moved all helm related in `charts/karpenter` and change the subsequent files (codegen and battletest)